src/lib/helpers.ts

```diff
@@ -5,6 +5,9 @@ import getConfig from 'next/config'
 import { i18n } from "next-i18next.config"
 import { BreadcrumbContent } from "./types"
 
+
+export const allowedEventTags = ['maahanmuuttajat', 'nuoret', 'info', 'koulutus', 'messut', 'neuvonta', 'rekrytointi', 'työpajat', 'digitaidot', 'etätapahtuma', 'palkkatuki', 'työnhaku']
```

Comment:
Yes, well we get these from Drupal but I have used this array for sorting the order of tags (because there is one) when those come from Elastic API. Syncing from Linked Events to Drupal I use the same array to check that only these tags get synced. In this frontend repo this array should actually be renamed and "allowed" part should be removed.